examples/dmd/heat_conduction_dmdc.cpp

```diff
@@ -20,13 +20,13 @@
 //   mpirun -np 8 heat_conduction_dmdc -s 1 -a 0.0 -k 1.0 -visit
 //
 // Output 1:
-//   Relative error of DMD temperature (u) at t_final: 0.5 is 0.0021716499
+//   Relative error of DMDc temperature (u) at t_final: 0.5 is 0.0019511173
```

Comment:
Can you check this?  I get 0.0021705658

Reply:
I still got 0.0019511173 while running on Quartz. Let me check it further.

Reply:
Fixed.